doc/Example-Gradle-Project/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
+
+# SPDX-FileCopyrightText: 2015 the original author or authors
+# SPDX-License-Identifier: Apache-2.0
+
+##############################################################################
+##
+##  Gradle start up script for UN*X
```

Comment:
Is this file copied from somewhere? If yes, we should document this for maintainability, for example because we might want to update this at some point.

Also, some licenses require that we need to document whether the file was copied unmodified or had changes, this also needs to be thought of.

Reply:
Thanks for the explanation. This is also not common for other build systems AFAIK. Typical ant, maven, cmake, npm, etc. projects all require you to have the build system installed locally.

And specifically, for an example project that is intended to show to Gradle users (which are likely to have Gradle already) how to use JavaSMT and is not really intended for anyone to check out and work on it, there seems even less reason to add this wrapper than for normal Gradle-based projects.

Reply:
This file is not supposed to be uploaded anyway. My bad.